example/ios/Podfile

```diff
@@ -3,12 +3,14 @@ require_relative '../node_modules/@react-native-community/cli-platform-ios/nativ
 
 source 'https://cdn.cocoapods.org'
 source 'https://github.com/bitmovin/cocoapod-specs.git'
+source 'https://github.com/react-native-tvos/react-native-tvos-podspecs.git'
 
-platform :ios, '12.4'
 install! 'cocoapods', :deterministic_uuids => false
 
-target 'BitmovinPlayerReactNativeExample' do
+def setup os
   inhibit_all_warnings!
+  platform os, '12.4'
```

Comment:
same here, why `12.4`?

Reply:
That's because the example app in [development](https://github.com/bitmovin/bitmovin-player-react-native/blob/development/example/package.json#L17) is using RN 0.69, which [requires iOS `12.4`](https://github.com/facebook/react-native/commit/982ca30de079d7e80bd0b50365d58b9048fb628f). So I just didn't change this value from what it was when I branched out.

In fact, this PR migrates the RN version to 0.68, because of [`react-native-tvos`](https://github.com/react-native-tvos/react-native-tvos/releases/tag/v0.68.2-6), so we could be using iOS `12.0` now. But it'd be nice to keep it as `12.4` because a new version of RN-tvos was released recently based on RN 0.69. And I should try to update the example app to use it later.

Reply:
I see. Is the min. deployment target for our RN lib documented on the README?

Reply:
No, that's not documented on README but it'd be pretty good to have actually. I think we could setup a `Requirements` section similar to what [RN has](https://github.com/facebook/react-native#-requirements).

Reply:
Yes. That would be great addition in a separate PR.